plugins/forc-index/src/ops/forc_index_revert.rs

```diff
@@ -0,0 +1,69 @@
+use crate::{cli::RevertCommand, utils::project_dir_info};
+use fuel_indexer_lib::manifest::Manifest;
+use reqwest::{
+    blocking::Client,
+    header::{HeaderMap, AUTHORIZATION},
+    StatusCode,
+};
+use serde_json::{to_string_pretty, value::Value, Map};
+use tracing::{error, info};
+
+pub async fn init(command: RevertCommand) -> anyhow::Result<()> {
+    let RevertCommand {
+        url,
+        path,
+        manifest,
+        ..
+    } = command;
+
+    let (_root_dir, manifest_path, _index_name) =
+        project_dir_info(path.as_ref(), manifest.as_ref())?;
+
+    println!("manifest_path: {:?}", manifest_path);
```

Comment:
Can we remove this as the user has to pass the manifest path in for the command to work?